### PR TITLE
Update images.yaml

### PR DESCRIPTION
**Pull request by JordanJordanov**
Set cloud.gardener.cnudie/responsibles for nginx-ingress-controller

**How to categorize this PR?**

/area ops-productivity
/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: imagevector/images.yaml [hunk 1]
```diff
@@ -72,6 +72,10 @@ images:
       confidentiality_requirement: 'low'
       integrity_requirement: 'low'
       availability_requirement: 'low'
+  - name: 'cloud.gardener.cnudie/responsibles'
+    value:
+    - type: 'githubUser'
+      username: 'shafeeqes'
```
Comment (acumino) [inline imagevector/images.yaml:78]: I'm not an expert in this area, but assigning a specific person for every image task seems inefficient. Shouldn't it be a team responsibility?

Comment (JordanJordanov) [reply imagevector/images.yaml:78]: Who is usually taking care for the CVEs in `nginx-ingress-controller`? Do you have dedicated people or it's a responibility for the current release responsible?

Comment (rfranzke) [reply imagevector/images.yaml:78]: Typically, the release responsible is the first contact person in case of vulnerabilities/compliance reports.

Comment (JordanJordanov) [reply imagevector/images.yaml:78]: OK, It doesn't really make sense to configure `cloud.gardener.cnudie/responsibles` for g/g components as you have your own schedule for release responsible who shall take care.

**Pull request closed without merging**